dapps/tests/app/config/blockchain.js

```diff
@@ -0,0 +1,84 @@
+// This file contains only the basic configuration you need to run Embark's node
+// For additional configurations, see: https://embark.status.im/docs/blockchain_configuration.html
+module.exports = {
+  // default applies to all environments
+  default: {
+    enabled: true,
+    client: "geth" // Can be geth or parity (default:geth)
+  },
+
+  development: {
+    clientConfig: {
+      miningMode: 'dev' // Mode in which the node mines. Options: dev, auto, always, off
+    },
+    "accounts": [
+      {
+        "mnemonic": "example exile argue silk regular smile grass bomb merge arm assist farm",
+        "balance": "5 ether"
+      }
+    ]
+  },
+
+  privatenet: {
+    // Accounts to use as node accounts
+    // The order here corresponds to the order of `web3.eth.getAccounts`, so the first one is the `defaultAccount`
+    // For more account configurations, see: https://embark.status.im/docs/blockchain_accounts_configuration.html
+    accounts: [
+      {
+        nodeAccounts: true, // Accounts use for the node
+        numAddresses: "1", // Number of addresses/accounts (defaults to 1)
+        password: "config/development/password" // Password file for the accounts
+      }
+    ],
+    clientConfig: {
+      datadir: ".embark/privatenet/datadir", // Data directory for the databases and keystore
+      miningMode: 'auto',
+      genesisBlock: "config/privatenet/genesis.json" // Genesis block to initiate on first creation of a development node
+    }
+  },
+
+  privateparitynet: {
+    client: "parity",
+    genesisBlock: "config/privatenet/genesis-parity.json",
+    datadir: ".embark/privatenet/datadir",
+    miningMode: 'off'
```

Comment:
I guess it should, but I made the refactor so that `clientConfig` is optional. In the end, all the configs inside `clientConfig` end up at the root anyway, because that's how `blockchain.js` handles it (that way I had very minimal changes to do to `blockchain.js`)
Let me know if you or anyone else has issues with that and I can make it obligatory